packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -5779,137 +5781,167 @@ namespace Tpetra {
   Kokkos::UnorderedMap<LocalOrdinal, size_t, typename Node::device_type>
```

Comment:
In the future, when we get rid of template parameters, we'll want to get `device_type` from RowGraph, not from Node.